audio-worklet/src/bitcrusher-demo.js

```diff
@@ -16,24 +16,39 @@
 class BitcrusherDemo {
   /**
    * Initalizes and modifies bitcrusher settings in response to GUI input.
-   * @param  {Number} bitDepth the number of bits for each sample
-   * @param  {Number} reduction the amount of sample rate reduction to apply
-   * @param  {Number} gain the volume of the output
+   * @param {AudioContext} context the Audio Context
+   * @param {String} containerId container for GUI elements
    */
-  constructor(containerId, bitDepth, reduction, gain) {
-    this.context_ = new AudioContext();
-    this.masterGain_ = new GainNode(this.context_, {gain: (gain || 0.5)});
-    this.bitcrusher_ = new Bitcrusher(this.context_, {
-      channelCount: 1,
-      bitDepth: bitDepth || 24,
-      reduction: reduction || 1
+  constructor(context, containerId) {
+    this.context_ = context;
+    this.masterGain_ = new GainNode(this.context_, {gain: 0.5});
+    
+    // Create a bit crusher with the audio worklet node.
+    this.bitcrusherAudioWorklet_ =
+        new AudioWorkletNode(this.context_, 'bitcrusher-audio-worklet');
+    this.paramBitDepth_ =
+        this.bitcrusherAudioWorklet_.parameters.get('bitDepth');
+    this.paramReduction_ =
+        this.bitcrusherAudioWorklet_.parameters.get('reduction');
+
+    // The script processor will have a delay relative to the audio worklet node
+    // proportional to the size of its buffersize. This delay can be heard as
+    // a glitch as the user switches node.
```

Comment:
Well, I am not sure if I can understand this comment. What do 'relative' and 'proportional' mean here w.r.t AudioWorkletNode?